Source/JavaScriptCore/offlineasm/x86.rb

```diff
@@ -94,71 +94,32 @@ def isMSVC
     $options.has_key?(:assembler) && $options[:assembler] == "MASM"
 end
 
-# TODO Cleanup unused isIntelSyntax paths
-def isIntelSyntax
-    $options.has_key?(:assembler) && $options[:assembler] == "MASM"
-end
-
 def register(name)
-    isIntelSyntax ? name : "%" + name
+    "%" + name
 end
 
 def offsetRegister(off, register)
-    isIntelSyntax ? "[#{off} + #{register}]" : "#{off}(#{register})"
+    "#{off}(#{register})"
 end
 
 def callPrefix
-    isIntelSyntax ? "" : "*"
+    "*"
 end
 
 def orderOperands(*operands)
-    (isIntelSyntax ? operands.reverse : operands).join(", ")
+    operands.join(", ")
 end
 
 def const(c)
-    isIntelSyntax ? "#{c}" : "$#{c}"
+    "$#{c}"
 end
 
 def const0x(c, kind)
-    if !isIntelSyntax
-        return "$0x#{c}"
-    end
-    case kind
-    when :half
-        "#{c}h"
-    when :quad
-        "#{c}h"
-    else
-        raise "bad kind for constant #{c} kind #{kind}"
-    end
+    return "$0x#{c}"
```

Comment:
nit: Here and immediately below, I suppose we can remove the `return` too.

Reply:
Done